_targets/build.project.armv7a9-zynq7000

```diff
@@ -96,12 +96,42 @@ b_build_target() {
 }
 
 
+b_write_cleanmarkers() {
+	local rootfs_file="$1"
+	local erase_sz="$2"
+	local part_sz="$3"
+
+	# 2 B : magic
+	# 2 B : node type
+	# 4 B : total len
+	# 4 B : crc
+	local cleanmarker="\x85\x19\x03\x20\x10\x00\x00\x00\x96\x58\xd1\xfe"
+
+	local rootfs_sz=$(du -b "$rootfs_file" | awk '{ print $1 }')
+	local blocks_cnt=$(((part_sz - rootfs_sz) / erase_sz))
```

Comment:
implicitly assuming rootfs_sz would be aligned to erase_sz (it will be due to how `mkfs.jffs2` and/or `sumtool` works), but for generic use case You should add `erase_sz -1` before division for `ceil` semantics

Reply:
I have added assert.